data-munging/spd_2020_salary_data.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+import logging
+from pathlib import Path
+
+import click
+import numpy as np
+import pandas as pd
+
+
+log = logging.getLogger()
+
+
+def main(id_path: Path, data: Path, output: Path):
+    log.info("Starting import")
+    df = pd.read_csv(data, usecols=["Name", "Base Pay", "Overtime"])
+    ids = pd.read_csv(
+        id_path,
+        usecols=["id", "first name", "last name"],
+    )
+    ids.columns = ["id", "last", "first"]
+    # Remove Jr, IV, II, III, etc.
+    df.loc[:, "Name"] = df.Name.replace(r" ?((Jr)|(II)|(III)|(IV))\.?", "", regex=True)
```

Comment:
Great question! The [pandas core team discourages its use](https://www.dataschool.io/future-of-pandas/#inplace), for a few reasons (see the link). It's also more explicit to show that we're updating the column with the `.loc` access.

Reply:
Ah yeah that definitely makes sense, thanks for the explainer. It definitely is a nice clear and concise way to explicitly state what column is being updated.

Glad I looked up the return type instead of copy/pasting it to the other uses of replace haha. Not being able to use it in a method chain does kinda suck.

Reply:
Yea there are definitely some good reasons there. I always used to do `inplace=True` but this feels more solid.